.gitignore

```diff
@@ -0,0 +1,7 @@
+.eggs/
+.tox/
+src/gocept.filestore.egg-info/
+src/gocept/__init__.pyc
+src/gocept/__pycache__/
+.coverage
+src/gocept/filestore/__pycache__/
```

Comment:
There are some newlines missing at the of a number of files. Maybe you can look into that/ or the editor settings.